exchanges/order/order_types.go

```diff
@@ -409,6 +403,28 @@ const (
 	MissingData
 )
 
+// TimeInForce enforces a standard for time-in-force values across the code base.
+type TimeInForce uint16
+
+// Is checks to see if the enum contains the flag
+func (t TimeInForce) Is(in TimeInForce) bool {
```

Comment:
PLease have a test for this and remove IsIOC method as this is more generic for that application.